src/index.html

```diff
@@ -12,18 +12,14 @@
       <a href="#" class="logo">
         <img src="./images/logo.png" alt="MOYO Logo">
       </a>
-      <nav>
-        <ul>
-          <li><a href="#">Home</a></li>
-
-          <li><a href="#">Shop</a></li>
-
-          <li><a href="#">About</a></li>
-
-          <li><a href="#" data-qa="hover">Laptops & computers</a></li>
-
-          <li><a href="#" class="is-active">Contact</a></li>
-        </ul>
+      <nav class="header-nav"></nav>
```

Comment:
Tutaj nav ci sie zamknał od razu po otwarciu i style dalej sie nei zaciągaja.
Łukasz jakbys mogł sprawdzic czy to w ogole wyglada jakkowiel u ciebie lokalnie przed wysłaniem do nas to oszczedziłoby nam duzo czasu. :))